src/usr/local/www/services_dhcpv6.php

```diff
@@ -227,10 +227,10 @@ function dhcpv6_apply_changes($dhcpdv6_enable_changed) {
 
 	// Note: if DHCPv6 Server is not enabled, then it is OK to adjust other parameters without specifying range from-to.
 	if ($_POST['enable']) {
-		$reqdfields = explode(" ", "range_from range_to");
-		$reqdfieldsn = array(gettext("Range begin"), gettext("Range end"));
-
-		do_input_validation($_POST, $reqdfields, $reqdfieldsn, $input_errors);
+		if (empty($_POST['range_from']) && empty($_POST['range_to']) &&
```

Comment:
The logic here seems wrong. It should be checking the mode and if it's not stateless then it should error if _either_ the range from or to is empty (not just both)

Reply:
right, fixed